internal/server/instance/drivers/driver_qemu.go

```diff
@@ -366,6 +366,12 @@ func (d *qemu) qmpConnect() (*qmp.Monitor, error) {
 // Callers should check that the instance is running (and therefore mounted) before calling this function,
 // otherwise the qmp.Connect call will fail to use the monitor socket file.
 func (d *qemu) getAgentClient() (*http.Client, error) {
+	// Check that the VM is in a state where the agent may be reachable.
+	status := d.statusCode()
+	if !d.isRunningStatusCode(status) || status == api.Frozen {
```

Comment:
Does this test mean that the comment above the function ("Callers should check that the instance is running ... before calling this function") is no longer relevant?